CHANGES.txt

```diff
@@ -5,6 +5,8 @@
  - Removed the disable_impressions configuration param.
  - Fixed an issue about Event properties were not sent to be.
  - Added ip_addresses_enabled to enable/disable sending MachineName and MachineIP headers when data is posted to Split Servers.
```

Comment:
Try to keep these first words in the past (we've not changed old items, but we should apply the new ways of the changelog) 
Ideally you would use the same four/five words to start the sentences, e.g.: Added, Removed, Updated, Fixed.

To give you an example: 
>  Resulting localhost data is now used indifferently from actual backend data by the SDK code.
could be
> Updated localhost mode to parse the mock data into Split objects to keep differences in operation to a minimum.